src/Hazelcast.Net.Examples/HazelcastOptionsBuilderExtensions.cs

```diff
@@ -27,18 +27,18 @@ public static class HazelcastOptionsBuilderExtensions
         /// <param name="builder">The options builder.</param>
         /// <param name="hazelcastLogLevel">The Hazelcast log level.</param>
         /// <returns>The options builder.</returns>
-        public static HazelcastOptionsBuilder WithConsoleLogger(this HazelcastOptionsBuilder builder, LogLevel hazelcastLogLevel = LogLevel.None)
+        public static HazelcastOptionsBuilder WithConsoleLogger(this HazelcastOptionsBuilder builder, LogLevel hazelcastLogLevel = LogLevel.Information)
         {
             return builder
                 .With("Logging:LogLevel:Default", "None")
                 .With("Logging:LogLevel:System", "Information")
                 .With("Logging:LogLevel:Microsoft", "Information")
-                .With("Logging:LogLevel:Hazelcast", hazelcastLogLevel.ToString())
-                .With((configuration, o) =>
+                //.With("Logging:LogLevel:Hazelcast", hazelcastLogLevel.ToString())
```

Comment:
Why comment out `"Logging:LogLevel:Hazelcast"`? It's also present in the documentation for `WithConsoleLogger` above.

Reply:
because at the moment in-memory options that you set with `With("option", "value")` take precedence over everything  - if we leave this here, you cannot override it with command-line options... but yea you're right it's documented... ok, adding a note to the doc for people to pay attention.